src/main/java/org/gridsuite/merge/orchestrator/server/repositories/MergeEntity.java

```diff
@@ -8,25 +8,44 @@
 
 import lombok.Getter;
 import lombok.ToString;
-import org.springframework.data.cassandra.core.mapping.PrimaryKey;
-import org.springframework.data.cassandra.core.mapping.Table;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.UUID;
 
 /**
  * @author Jon harper <jon.harper at rte-france.com>
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com
  */
+@Entity
 @Getter
 @ToString
-@Table("merge")
-public class MergeEntity {
+@Table(name = "merge")
+@IdClass(MergeEntityKey.class)
+public class MergeEntity implements Serializable {
+
+    @Id
```

Comment:
we can use the EmbeddedId Annotation instead

Reply:
I choose the IdClass annotation solution for composite primary key here, just because select requests are less verbose with this solution ....
We can effectively also use the EmbeddedId solution ... But, is it really needed ?